test/libcommonserver/log/testlog.cpp

```diff
@@ -51,4 +47,78 @@ void TestLog::testLog() {
 
     CPPUNIT_ASSERT(true);
 }
+
+void TestLog::testLogRollingAndExpiration(void) {
+    // Clear the log directory
+    IoError ioError = IoErrorSuccess;
+    IoHelper::DirectoryIterator dirIt(_logDir, false, ioError);
+
+    bool endOfDirectory = false;
+    DirectoryEntry entry;
+    while (dirIt.next(entry, endOfDirectory, ioError) && !endOfDirectory && ioError == IoErrorSuccess) {
+        if (entry.path().filename().string() == Log::instance()->getLogFilePath().filename().string()) {
+            continue;
+        }
+        IoHelper::deleteDirectory(entry.path(), ioError);
+    }
+
+    testLargeLogRolling();
+    testExpiredLogFiles();
```

Comment:
Wouldn't it better to run those tests separetly?
And therefor, moving the clearing of the log directory in a dedicated function to be called at the begining of each test